docs/ChannelValue.md

```diff
@@ -1,42 +1,54 @@
 
 Icons are passed in as a child element of the ChannelValue component.
 
 <div style="text-align:center; margin-bottom:20px">
-<img width="100%" style="max-width: 600px" alt="Channel Value in Hero" src="./images/heroes.png">
-<hr/>
-<img width="100%" style="max-width: 600px" alt="Channel Value in a list item" src="./images/channelValue.png">
+    <img width="100%" style="max-width: 600px" alt="Channel Value in Hero" src="./images/heroes.png">
+    <hr/>
+    <img width="100%" style="max-width: 600px" alt="Channel Value in a list item" src="./images/channelValue.png">
 </div>
 
 ## Usage
 
+<div style="width: 100%; text-align: center">
+    <img width="100%" style="max-width: 400px" alt="Channel Value Anatomy" src="./images/channelValueAnatomy.png">
+</div>
+
 ```typescript
 // app.module.ts
 import { ChannelValueModule } from '@pxblue/angular-components';
 ...
 imports: [
     ChannelValueModule
-  ],
+],
 ```
 
-```typescript
+```html
 // your-component.html
-<pxb-channel-value class="content" value="65" units="%" [prefix]="true" fontSize="10px" >
-  <mat-icon >trending_up</mat-icon>
+<pxb-channel-value class="content" value="65" units="%" [prefix]="true" fontSize="10px">
+    <mat-icon>trending_up</mat-icon>
 </pxb-channel-value>
 ```
 
 ## API
 
+Parent element (`pxb-channel-value`) attributes:
+
 <div style="overflow: auto;">
 
-| Prop Name   | Description                                    | Type                     | Required | Default   |
-|-------------|------------------------------------------------|--------------------------|----------|-----------|
-| value       | The the value (bold text) to display           | `string` &vert; `Number` | yes      |           |
-| units       | The text to display for the units (light text) | `string`                 | no       |           |
-| prefix      | Show units before the value                    | `boolean`                | no       | false     |
-| fontSize    | The size of the font                           | `string`                 | no       | 'inherit' |
-| color       | The color of the font                          | `string`                 | no       | 'inherit' |
+| Attributes | Description                                    | Type              | Required | Default   |
+| ---------- | ---------------------------------------------- | ----------------- | -------- | --------- |
+| fontSize   | The size of the font                           | `string`          | no       | 'inherit' |
+| prefix     | Show units before the value                    | `boolean`         | no       | false     |
+| units      | The text to display for the units (light text) | `string`          | no       |           |
+| value      | The the value (bold text) to display           | `string | number` | yes      |           |
 
 </div>
+
+Child element:
```

Comment:
We should have consistent language across readmes for this...some say "Child element" others say "Child element with attributes"...Is there a distinction that we should explain? Does this particular child need to have an attribute to be picked up?

Reply:
Child element:
Usually only one child element is needed, no attributes needed

Child element with attributes:
Multiple child elements are picked up by the component, and they all have attributes

Tell me how you would like to word these.

Reply:
Okay that makes sense. Maybe we just leave the attribute column blank for the case when there's no specific selector/attribute used to transclude the child.

Reply:
I am using "(child)" right now